spec.html

```diff
@@ -19,49 +19,48 @@ <h1>The Intl.Locale Constructor</h1>
     <emu-clause id="sec-apply-options-to-tag" aoid=ApplyOptionsToTag>
       <h1>ApplyOptionsToTag( _tag_, _options_ )</h1>
       <p>
-        The following algorithm refers to <a href="https://tools.ietf.org/html/rfc5646#section-2.1">RFC 5646's Language-Tag grammar</a>.
+        The following algorithm refers to <a href="http://www.unicode.org/reports/tr35/#Identifiers">UTS 35's Unicode Language and Locale Identifiers grammar</a>.
```

Comment:
Nit: this seems to work as https, so let's use that.

Reply:
DONE